container/demo/README.md

```diff
@@ -0,0 +1,276 @@
+
+```bash
+    # Move to the container folder (ks-nav/container)
+    cd container
+    # Create an output folder for ksnav images
+    mkdir ./demo/ksbuild/out_img
+    # Copy the ksnav config files to the /app folder 
+    cp ksnav_kdb_local.json ./demo/ksbuild/app/.
+    cp ksnav_nav_local.json ./demo/ksbuild/app/.
+```
+ks-nav can be run through a CLI, or through a web interface. We'll go through both options in this demo. 
+
+#### 2.2.1 Run ks-nav from CLI
+
+```bash
+    # OPTION 1: RUN KS-NAV FROM CLI 
+    # These steps should start from the ks-nav/container directory
+    # Comment out the ENTRYPOINT in the Dockerfile to disable the launch of the web server
+    cat Dockerfile | \
+        sed -r "s|ENTRYPOINT|#ENTRYPOINT |g" | tee Dockerfile_clidemo
+    # Build the ksnav container 
+    podman build -t ksnav-img -f Dockerfile_clidemo . --build-arg _EN_DEMO=true
+    # Run it in interractive mode, mapping the linux image and the output flder
+    podman run -it --name ksnav-cont -p 5432:5432 -p 8080:8080 \
+     -v $(pwd)/demo/ksbuild/app:/app:z \
+     -v $(pwd)/demo/ksbuild/out_img:/out_img:z \
+     ksnav-img sh
+    # Start the database service (if it wasn't already running)
+    su posgres 
+    pg_ctl start -D /var/lib/pgsql/data/only/
+    exit 
+    # Build the database for ks-nav
+    cd /app
+    nav-db-filler ksnav_kdb_local.json    
+    # Run the Source code Navigator which generates call tree graphs. 
+    # The graphs generated are redirected to a file in the ./demo/ksbuild/out_img folder
+    nav -f ksnav_nav.json > /out_img/run_init_process
+```
+
+Folder **demo/ksbuild/out_img** is now populated with a JPEG image *run_init_process*.
+
+#### 2.2.2 Run ks-nav from Web User Interface
+
+We will generate the same graph from the CLI section; this time, from the web user interface. 
+```bash
+    # Move to the container directory (ks-nav/container)
+    cd container
+    podman stop ksnav-cont; podman rm ksnav-cont
+    #Build the Docker image for ksnav as shown below
+    podman build -t ksnav-img -f Dockerfile . --build-arg _EN_DEMO=true
+    # Run it in interractive mode, mapping the linux image and the output folder
+    podman run -d --name ksnav-cont -p 5432:5432 -p 8080:8080 \
+     -v $(pwd)/demo/ksbuild/app:/app:z \
+     -v $(pwd)/demo/ksbuild/out_img:/out_img:z \
+     ksnav-img sh
+    # The web server should now be running
+```
+Open a web browser at: http://localhost:8080 and click on `Acquire`
+The script will build the kern-db database. This could take a while.
+Once the command is finished executing, go back to http://localhost:8080 and click on `Explore`
+Fill out the form as follow: 
+
+```bash
+    start_symbol = run_init_process
+    instance = 1
+    display mode = Functional
+    depth = 2
+    Click on 'Generate Image'
+```
+
+At the end of this step, the same image from the CLI step is displayed on the web interface.
+
+## 3 Detailed Tools Configurations
```

Comment:
Move parts into the existing docs and then reference here

Reply:
Separate early commit before actual demo (could be merged on it's own to help support the tool)